### ISSUE TITLE
Invite Issue

### ISSUE THREAD

**Issue by michaelmalonenz** [michaelmalonenz/aurelia-dragula#43]
@n2liquid Yeah, I gave up on the idea - it didn't get merged (as you might have noticed), but I liked the library enough to port it to Aurelia, as you might see here.

**Comment by n2liquid** [comment]
Hey, Michael. Thanks for letting me know about this project, it looks interesting. Are all tests passing? If you got all of them to pass and you remove the Aurelia-specific bits, upstream devs might want to have it merged. I think they didn't merge it because of the broken tests and because they didn't understand the problem. That behavior is clearly, positively a bug in Dragula. If the operation is cancelled, the DOM should be returned to its original state, and that's not happening. I would think they have an interest in getting that fixed at least...

**Comment by michaelmalonenz** [comment]
I got confused about the tests not running in that initial PR - I thought I had broken something unrelated.  As it turns out, they were already broken, see [this PR](https://github.com/bevacqua/dragula/pull/352) for more info.

I got a lot of silence from the maintainer whenever I put up PRs, which is his right - he doesn't owe me anything - but it became a lower barrier to entry for me to fork and make it friendlier for Aurelia.  As part of my fork, I ported the language from es5 to es2016 - even if I were to separate the Aurelia specific parts into a plain wrapper, I couldn't do a basic merge, now, anyway. 

Also, I do aurelia-specific things in the depths of the library (I pass the ViewModel instance(s) along with the various event callbacks).
